services/github/github-actions-workflow-status.service.js

```diff
@@ -0,0 +1,85 @@
+import Joi from 'joi'
+import { isBuildStatus, renderBuildStatusBadge } from '../build-status.js'
+import { BaseSvgScrapingService } from '../index.js'
+import { documentation } from './github-helpers.js'
+
+const schema = Joi.object({
+  message: Joi.alternatives()
+    .try(isBuildStatus, Joi.equal('no status'))
+    .required(),
+}).required()
+
+const queryParamSchema = Joi.object({
+  event: Joi.string(),
+  branch: Joi.string(),
+}).required()
+
+const keywords = ['action', 'actions']
+
+export default class GithubActionsWorkflowStatus extends BaseSvgScrapingService {
+  static category = 'build'
+
+  static route = {
+    base: 'github/actions/workflow/status',
+    pattern: ':user/:repo/:workflow+',
```

Comment:
A workflow file can be in a subdirectory. This means `:workflow` can contain a slash, so I've made this `:workflow+` and converted branch to a query param.